libchisel/Cargo.toml

```diff
@@ -11,6 +11,6 @@ keywords = ["webassembly", "wasm", "blockchain", "ethereum"]
 edition = "2018"
 
 [dependencies]
-parity-wasm = "^0.35.6"
 rustc-hex = "1.0"
 byteorder = "1.2.4"
+parity-wasm = "^0.35.0" 
```

Comment:
Why did you go back from 0.35.6 to 0.35.0? I think there was a reason requiring 0.35.6 as the minimum.